rules/integrations/aws/defense_evasion_escalation_aws_suspicious_saml_activity.toml

```diff
@@ -1,77 +0,0 @@
-[metadata]
-creation_date = "2021/09/22"
-integration = ["aws"]
-maturity = "production"
-updated_date = "2024/05/21"
```

Comment:
Should this not be deprecated like all other rules?

Reply:
It was just renamed, the UUID is maintained in the one below

Reply:
Ah.. ok